Description: Modify the source paths to build the Ant tasks with eclipse-debian-helper
Author: Emmanuel Bourg <ebourg@pache.org>
Forwarded: not-needed
--- a/bundles/org.eclipse.equinox.p2.repository.tools/build.properties
+++ b/bundles/org.eclipse.equinox.p2.repository.tools/build.properties
@@ -11,7 +11,7 @@
 # Contributors:
 #     IBM Corporation - initial API and implementation
 ###############################################################################
-source.. = src/
+source.. = src/,src_ant/
 output.. = bin/
 bin.includes = META-INF/,\
                .,\
--- a/bundles/org.eclipse.equinox.p2.jarprocessor/build.properties
+++ b/bundles/org.eclipse.equinox.p2.jarprocessor/build.properties
@@ -11,7 +11,7 @@
 #  Contributors:
 #     IBM Corporation - initial API and implementation
 ###############################################################################
-source.. = src/
+source.. = src/,src_ant/
 output.. = bin/
 bin.includes = META-INF/,\
                .,\
--- a/bundles/org.eclipse.equinox.p2.director.app/build.properties
+++ b/bundles/org.eclipse.equinox.p2.director.app/build.properties
@@ -11,7 +11,7 @@
 # Contributors:
 #     IBM Corporation - initial API and implementation
 ###############################################################################
-source.. = src/
+source.. = src/,src_ant/
 source.ant_tasks/director-ant.jar=src_ant/
 output.. = bin/
 bin.includes = META-INF/,\
--- a/bundles/org.eclipse.equinox.p2.metadata.repository/build.properties
+++ b/bundles/org.eclipse.equinox.p2.metadata.repository/build.properties
@@ -11,7 +11,7 @@
 # Contributors:
 #     IBM Corporation - initial API and implementation
 ###############################################################################
-source.. = src/
+source.. = src/,src_ant/
 output.. = bin/
 bin.includes = META-INF/,\
                .,\
--- a/bundles/org.eclipse.equinox.p2.publisher.eclipse/build.properties
+++ b/bundles/org.eclipse.equinox.p2.publisher.eclipse/build.properties
@@ -11,7 +11,7 @@
 # Contributors:
 #     IBM Corporation - initial API and implementation
 ###############################################################################
-source.. = src/
+source.. = src/,src_ant/
 bin.includes = META-INF/,\
  				.,\
                plugin.properties,\
